### DISCUSSION TITLE
nRF24/PA CW receiver

### DISCUSSION THREAD

**Discussion by g8kuw-tv** [jgromes/RadioLib#241] [Q&A]
First, thanks for the superb library - I've been having loads of lock-down fun messing with 433MHz, RF95 LoRa and nRF24, RTTY, Morse etc! I now want try some mobile field strength measurement from RF24+PA to RF24 direct, so I set up an ISM 2401MHz (<0 dBm) beacon going that is just TX-ing my callsign and QRA locator. I can hear it fine on a shack-based SDR like a HackRF, but I'd like to use another RF24 as a portable handheld 2.4GHz receiver (RF24 + Arduino + SSD1306 OLED + battery pack). 

Is it possible to read an actual, or approximate RF24 RSSI measurement from this library?

**Comment by jgromes** [comment]
Unfortunately, nRF24 only contains a single-bit RSSI information - RSSI above or below some predefined threshold. That might be enough for CW decoding, but I haven't tried it.

Overall nRF24 is probably the worst of all of the supported modules for this sort of ham-radio hackery.

**Reply by g8kuw-tv** [reply]
Good to know, thanks.
